Sources/HTTP/Models/Server/HTTP+Server.swift

```diff
@@ -25,11 +25,18 @@ public final class Server<
     public let host: String
     public let port: Int
     public let securityLayer: SecurityLayer
-
-    public init(host: String = "0.0.0.0", port: Int = 8080, securityLayer: SecurityLayer = .none) throws {
+    public let middleware: [Middleware]
+
+    public init(
+        host: String = "0.0.0.0",
+        port: Int = 8080,
+        securityLayer: SecurityLayer = .none,
+        middleware: [Middleware] = []
```

Comment:
I think maybe client should use a default middleware parameter here as well, ie: `middleware: [Middleware] = Client.defaultMiddleware`